### PR TITLE
Added database wipe function

### PR DESCRIPTION
**Pull request by OscarB28**
Added the function to clear the database at the end

### CODE REVIEW

File: rds_s3_pipeline/daily_extract.py [hunk 1]
```diff
@@ -48,6 +49,18 @@ def extract_from_rds(db_conn: Connection) -> list[tuple]:
     return res
 
 
+def wipe_from_rds(db_conn: Connection) -> None:
```
Comment (IB64) [inline rds_s3_pipeline/daily_extract.py:52]: clean function

File: rds_s3_pipeline/daily_extract.py [hunk 2]
```diff
@@ -95,9 +110,18 @@ def convert_to_csv_and_upload(plant_conditions: list[dict], s3_client: client, b
...
         list_of_conditions = extract_from_rds(conn)
+        print("Creating dictionaries from data...")
         plant_conditions_dicts = create_condition_dicts(list_of_conditions)
+        print("Converting to .csv and uploading...")
         convert_to_csv_and_upload(
             plant_conditions_dicts, s3, "c9-queenbees-bucket")
+        print("Deleting old data...")
+        wipe_from_rds(conn)
+
+        print(f"Extraction complete --- {perf_counter() - time_to_run}s.")
```
Comment (caithomas6789) [inline rds_s3_pipeline/daily_extract.py:124]: Good idea to log progress of the script.

**Review by IB64 — APPROVED** [review]
Nice

File: rds_s3_pipeline/daily_extract.py [hunk 3]
```diff
@@ -87,6 +100,8 @@ def convert_to_csv_and_upload(plant_conditions: list[dict], s3_client: client, b
 
 if __name__ == "__main__":
 
+    time_to_run = perf_counter()
```
Comment (caithomas6789) [inline rds_s3_pipeline/daily_extract.py:103]: Good idea to time certain processes in the script.

**Pull request merged**